code/game/turfs/simulated/shuttle.dm

```diff
@@ -44,7 +44,10 @@
 /turf/simulated/wall/shuttle/ex_act(severity)
 	return
 
-/turf/simulated/wall/shuttle/mech_drill_act(severity)
```

Comment:
You need to keep this otherwise they can be drilled by mechs.

Reply:
it just calls dismantle_wall on its parent type, which i blanked to a return anyways